src/cascadia/TerminalSettingsEditor/KeyChordListener.xaml

```diff
@@ -0,0 +1,18 @@
+<UserControl x:Class="Microsoft.Terminal.Settings.Editor.KeyChordListener"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             mc:Ignorable="d">
+
+    <!--  Use PreviewKeyDown to prevent ctrl+a from selecting  -->
+    <TextBox x:Name="KeyChordTextBox"
+             x:Uid="KeyChordListener_KeyChordTextBox"
+             IsReadOnly="True"
+             IsSpellCheckEnabled="False"
+             IsTabStop="True"
+             LosingFocus="KeyChordTextBox_LosingFocus"
+             PreviewKeyDown="KeyChordTextBox_KeyDown"

```

Comment:
FYI the way this is set up, styles on the parent control do not propagate into the TextBox. This is probably _okay_ for this specific control.

Reply:
Yeah, we'll come back to this if/when we want to use the key chord listener somewhere else. Wasn't sure how much we wanted to standardize within the control vs expose.